App.js

```diff
@@ -0,0 +1,37 @@
+import { StatusBar } from 'expo-status-bar';
+import { StyleSheet, Text, View } from 'react-native';
+import {
+  useFonts,
+  IBMPlexSerif_100Thin,
+  IBMPlexSerif_200ExtraLight,
+  IBMPlexSerif_300Light,
+  IBMPlexSerif_400Regular,
+  IBMPlexSerif_500Medium,
+  IBMPlexSerif_600SemiBold,
+  IBMPlexSerif_700Bold,
+} from '@expo-google-fonts/ibm-plex-serif';
+import StackNavigator from './src/Navigators/StackNavigator';
+
+export default function App() {
+
+  let [fontsLoaded] = useFonts({
+    IBMPlexSerif_100Thin,
+    IBMPlexSerif_200ExtraLight,
+    IBMPlexSerif_300Light,
+    IBMPlexSerif_400Regular,
+    IBMPlexSerif_500Medium,
+    IBMPlexSerif_600SemiBold,
+    IBMPlexSerif_700Bold,
+  });
+
+  if (!fontsLoaded) {
+    return null;
```

Comment:
We should provide fallback font options here, not return null. Check with Subhasis regarding this.

Reply:
So, what happens here when fonts are not loaded is that, it takes some buffer time to load them. If they cannot be loaded, it automatically displays the app in default text as Subhasis told in the discord server, so no need to add a fallback font. But, I have incorporated an additional change, where instead of loading on a blank page, it would display 'Loading..." Probably we already have a Qur8 logo image that we can use in its place when the MVP has finished.

Reply:
As per my understanding it's returning null, it's not returning Stacknavigator where all your component are, so it won't even know what your components are, so it will not load anything with default font. You should set fontFamily: 'System' in case font is not loaded.
You can test this behaviour by deleting the fonts from your assets directory, and let me know what happens.